gadgets/cmr/config/LandmarkDetection/CMR_RTCine_LAX_AI_Complex.xml

```diff
@@ -0,0 +1,198 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Feels like this config is very close to just a generic complex SNR scaled recon. Should it just be called that? Is it specific to the AI it will be used with?